docs/pages/enroll-resources/database-access/enroll-google-cloud-databases/alloydb.mdx

```diff
@@ -0,0 +1,307 @@
+
+![Grant permissions to user Service Account](../../../../img/database-access/guides/alloydb/user-service-account-permissions.png)
+
+Click "Done".
+
+### Grant access to the service account
+
+The Teleport Database Service must be able to impersonate this service account.
+Navigate to the "alloydb-user" service account overview page and select the
+"Principals with Access" tab:
+
+![Select Principals with Access Tab](../../../../img/database-access/guides/alloydb/user-service-account-principals-with-access.png)
+
+Click "Grant Access" and add the "teleport-db-service" principal ID.
+Select the "Service Account Token Creator" role and save the change:
+
+![Grant Service Account Token Creator to Database Service](../../../../img/database-access/guides/alloydb/user-service-account-grant-access.png)
+
+<Admonition type="note" title="Service account permissions">
+  The "Service Account Token Creator" IAM role includes more permissions than
+  the Teleport Database Service needs. To further restrict the service 
+  account, you can create a role that includes only the following permission:
+  ```ini
+  # Used to generate IAM auth tokens when connecting to a database instance.
+  iam.serviceAccounts.getAccessToken
+  ```
+</Admonition>
+
+## Step 3/9. Configure your AlloyDB database
+
+Teleport uses [IAM database authentication](https://cloud.google.com/alloydb/docs/database-users/manage-iam-auth)
+with AlloyDB instances.
+
+Navigate to your instance settings and enable the `alloydb.iam_authentication` flag under Advanced Configuration Options section.
+
+![Enable IAM Authentication](../../../../img/database-access/guides/alloydb/flag-iam-authentication-on.png)
+
+### Create a database user
+
+Now go back to the Users page of your AlloyDB instance and add a new user
+account. In the sidebar, choose "Cloud IAM" authentication type and add the
+"alloydb-user" service account that you created in
+[the second step](#step-29-create-a-service-account-for-a-database-user):
+
+![Add AlloyDB User Account](../../../../img/database-access/guides/alloydb/add-user-account.png)
+
+Press "Add" and your Users table should look similar to this:
+
+![AlloyDB User Accounts Table](../../../../img/database-access/guides/alloydb/user-account-added.png)
+
+See [Manage IAM authentication](https://cloud.google.com/alloydb/docs/database-users/manage-iam-auth)
+in Google Cloud documentation for more info.
+
+## Step 4/9. Install Teleport
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 5/9. Configure the Teleport Database Service
+
+### Create a join token
+
+(!docs/pages/includes/tctl-token.mdx serviceName="Database" tokenType="db" tokenFile="/tmp/token"!)
+
+### Generate Teleport config
+
+Provide the following information and then generate a configuration file for the
+Teleport Database Service:
+
+- <Var name="example.teleport.sh:443" /> The host and port of your Teleport
```

Comment:
@ptgott i recall there were some discussions to use `teleport.example.com:443` for Teleport domain name, and only use `example.teleport.sh:443` for cloud-specific stuff. could you confirm?

Reply:
it creates an issue where the same doc uses `teleport.example.com:443` and `example.teleport.sh:443` in difference sections of the page. especially considering now we are trying to use `Var`.